drivers/wdm/audio/backpln/portcls/irpstream.cpp

```diff
@@ -185,15 +234,23 @@ CIrpQueue::AddMapping(
         if (m_Descriptor->DataFlow == KSPIN_DATAFLOW_IN)
         {
             // irp sink
-            StreamData->TotalStreamData += Header->DataUsed;
+            HeaderLength = Header->DataUsed;
         }
         else
         {
             // irp source
-            StreamData->TotalStreamData += Header->FrameExtent;
+            HeaderLength = Header->FrameExtent;
         }
 
-        /* move to next header */
+        // increment available data
+        TotalStreamData += HeaderLength;
+
+        // append page count
+        StreamPageCount += ADDRESS_AND_SIZE_TO_SPAN_PAGES(
+            MmGetMdlByteOffset(Mdl), HeaderLength);
```

Comment:
```suggestion
        StreamPageCount += ADDRESS_AND_SIZE_TO_SPAN_PAGES(
                               MmGetMdlByteOffset(Mdl), HeaderLength);
```

Reply:
How did you arrive at that indentation depth it seems arbitrary?

Reply:
That was to put the function/macro's parameters below the other function-related stuff (usually we do:
```c
returnValue = FunctionCall(first_param,
                           ...);
```
when the first line is not too long already).

Reply:
Ok I understand and appreciate the aesthetic of keeping the function arguments further to the right that the function name. (I still think full alignment as given in your example is disgusting, but the suggestion is good)